File: Easy/C/The Running Race.c
#include <inttypes.h>
#include <stdio.h>

int main() {
    uint8_t D, X, Y;
    scanf("%" SCNu8 " %" SCNu8 " %" SCNu8, &D, &X, &Y);
    if(X == Y)
        printf("Draw");
    else if(X > Y)
        printf("Alex");
    else
        printf("Ryan");

    return 0;
}

File: Easy/Rust/The Running Race.rs
use std::io;

fn main() {
    let mut buf = String::new();
    io::stdin().read_line(&mut buf).unwrap();
    let mut buf = buf.split_whitespace();
    let D: u8 = buf.next().unwrap().parse().unwrap();
    let X: u8 = buf.next().unwrap().parse().unwrap();
    let Y: u8 = buf.next().unwrap().parse().unwrap();
    match X > Y {
        true => print!("Alex"),
        false => match X < Y {
            true => print!("Ryan"),
            false => print!("Draw"),
        },
    };
}
